sass/routes/_admin-console.scss

```diff
@@ -946,3 +946,12 @@
         max-width: none;
     }
 }
+
+#admin-logs-content {
```

Comment:
@Willyfrog was talking about this, @agnivade instead of using id here as css selector, can you use class

```
.admin-console__content {

}
```

Reply:
Sorry important is an issue, Can you change this to instead

```
.admin-console .admin-console__content.admin-logs-content {

}

```

Reply:
Done. Thanks.